dev-tools/composer.json

```diff
@@ -3,7 +3,7 @@
         "php": "^7.1"
     },
     "require-dev": {
-        "localheinz/composer-normalize": "^1.0.0",
+        "localheinz/composer-normalize": "^1.1.0",
```

Comment:
let's write it as `^1.1` (no need for PATCH number) and merge it once, without future updates for each release, as long as there is nothing to update on our side (like change of config or sth)

Reply:
What if we would remove these 4 deps from `composer.json` and install them via `composer require`? That wouldn't slow things, would it?

Reply:
if we would do that, what would be the benefit ?

Reply:
We'd have latest stable versions of those packages (I assume there won't be any conflicts between them) and will avoid bumping version constraint - `composer-require-checker` has already `1.1` tag, `mi-schi/phpmd-extension` has `4.3` and `phpmd/phpmd` has `2.6`.

Reply:
I didn't want to `composer require localheinz/composer-normalize:^1`, but simply `composer require localheinz/composer-normalize` to cover future major version bumps - I believe we have very little chance to stuck on BC break of these packages.

Reply:
then I would say no to that.
don't install tool without providing major version.

you may hope that if you currently have v1, that v2 won't provide BC break that we need to consider, but can you say the same between v1 and v5 ?